liferay-echo/liferay-portal
# LPS-184264 Remove select_theme.jsp and SelectThemeDisplayContext.java in order to use an Item Selector

Motivation
=======
The motivation behind this ticket is to use the generic Item Selector interface instead of a custom item selector to select design themes for widget pages.
[LPS-184264](https://issues.liferay.com/browse/LPS-184266)

Proposed Solution
========
What we did to solve this problem is to remove the select_theme.jsp with its context provider and use the generic item selector by creating an implementation for the layout admin module.

Steps to Verify
========
1. Go to the Product Menu -> Site Builder -> Pages
2. Create a widget page and configure it by clicking on the ellipsis button and the the configure button
3. Go to Design view
4. At the Theme section select the "Define a custom theme for this page" check input.
5. The button Change Current Theme appears, click on it.
6. A modal should be opened Check that the selector works fine.
